### PR TITLE
Minor refresh

### PR DESCRIPTION
**Pull request by chevdor**
This PR fixes a few linter grumbles and mainly adds keywords for a better discovery.
I wish I found this project/repo earlier :)

### CODE REVIEW

File: .github/workflows/quick-check.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+      - name: Cargo test
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+
+      - name: Cargo check
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
```
Comment (Keats) [inline .github/workflows/quick-check.yml:66]: why add that compared to the current pipeline?

File: Cargo.toml [hunk 2]
```diff
@@ -1,24 +1,25 @@
 [package]
-name = "kickstart"
-version = "0.2.1"
-description = "A simple way to get started with a project"
 authors = ["Vincent Prouillet <hello@vincentprouillet.com>"]
-license = "MIT"
+description = "A simple way to get started with a project by scaffolding from a template powered by the tera engine"
```
Comment (Keats) [inline Cargo.toml:3]: Tera with a capital T

**Review by Keats — CHANGES_REQUESTED** [review]
Looks good overall but I don't get why you added the GH actions file, Azure pipelines is already pretty fast.

**Comment by Keats** [comment]
Any big features missing for your usage?

File: .github/workflows/quick-check.yml (hunk 1 above)
Comment (chevdor) [reply .github/workflows/quick-check.yml:66]: I did not notice your pipeline at first :)
I also cannot run your pipeline in my fork (beside when opening a PR) so the github workflow makes it easier to check on my end.

I can remove it from the PR if you prefer.

**Comment by chevdor** [comment]
> Any big features missing for your usage?

Not at that point but I did not test extensively yet. I will happily report back or open a PR if so. Thanks for asking !

File: .github/workflows/quick-check.yml (hunk 1 above)
Comment (Keats) [reply .github/workflows/quick-check.yml]: yes please, no need for 2 CI setup

**Comment by chevdor** [comment]
> Any big features missing for your usage?

Since you asked :) https://github.com/Keats/kickstart/issues/48

**Review by Keats — APPROVED** [review]

**Pull request merged**